docs/how_to/storage/save_load.md

```diff
@@ -42,5 +42,41 @@ graph = load_graph_from_storage(storage_context, root_id="<root_id>") # loads gr
 
 Here's the full [API Reference on saving and loading](/reference/storage/indices_save_load.rst).
 
+## Using a remote backend
 
+By default, LlamaIndex uses a local filesystem to load and save files. However, you can override this by passing a `fsspec.AbstractFileSystem` object.
 
+```python
+import dotenv
+import s3fs
+import os
+dotenv.load_dotenv("../../../.env")
+
+# set up s3fs
+AWS_KEY = os.environ['AWS_ACCESS_KEY_ID']
+AWS_SECRET = os.environ['AWS_SECRET_ACCESS_KEY']
+R2_ACCOUNT_ID = os.environ['R2_ACCOUNT_ID']
+
+assert AWS_KEY is not None and AWS_KEY != ""
+
+s3 = s3fs.S3FileSystem(
+   key=AWS_KEY,
+   secret=AWS_SECRET,
+   endpoint_url=f'https://{R2_ACCOUNT_ID}.r2.cloudflarestorage.com',
+   s3_additional_kwargs={'ACL': 'public-read'}
+)
+
+# load documents
+documents = SimpleDirectoryReader('../../../examples/paul_graham_essay/data/').load_data()
+print(len(documents))
+index = GPTVectorStoreIndex.from_documents(documents, fs=s3)
```

Comment:
what is the purpose of passing the `fs` in here?

Reply:
in fact I don't see implementation that enables this. not sure if I missed how this is handled.

Reply:
removing